src/v/kafka/server/tests/quota_manager_bench.cc

```diff
@@ -135,3 +138,327 @@ PERF_TEST_CN(throughput_group, test_quota_manager_off_unique) {
       .use_unique = true,
     });
 }
+
+struct latency_test_case {
+    bool is_new_client;
+    bool is_produce_not_fetch;
+    int n_other_clients;
+    bool on_shard_0;
+    bool no_quotas;
+};
```

Comment:
nit: probably a nice clean up here would be using an `ss::bool_class`

Reply:
I'm creating the structs by assigning booleans to the fields and I am never passing the raw booleans to functions, so there's no part of the code where I could easily confuse which boolean means what - at least for now... So I think for now I am going to stick with using `bool` to keep it simple.